docs-web/configuration/transportserver-resource.md

```diff
@@ -310,6 +310,6 @@ The [ConfigMap](/nginx-ingress-controller/configuration/global-configuration/con
 ## Limitations
 
 As of Release 1.7, the TransportServer resource is a preview feature. Currently, it comes with the following limitations:
```

Comment:
I think it make sense to update 1.9 here
```suggestion
As of Release 1.9, the TransportServer resource is a preview feature. Currently, it comes with the following limitations:
```

Reply:
went with just
```
The TransportServer resource is a preview feature. Currently, it comes with the following limitations:
```